lib/kubernetes-deploy/kubernetes_resource/pod.rb

```diff
@@ -120,5 +140,69 @@ def display_logs
 
       @already_displayed = true
     end
+
+    class Container
+      STATUS_SCAFFOLD = {
```

Comment:
Not really, since it is empty? It's not there to provide fallback data, but rather to make it easier not to 💥  if the json we got doesn't contain all the keys we use.